### PR TITLE
chore(tests): Extend post-deploy tests

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

**Review by madlabman — APPROVED** [review]

File: test/fork/deployment/PostDeployment.t.sol [hunk 1]
```diff
@@ -24,12 +24,14 @@ import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/I
...
 contract CSModuleDeploymentTest is Test, Utilities, DeploymentFixtures {
     DeployParams private deployParams;
+    uint256 adminsCount;
 
     function setUp() public {
         Env memory env = envVars();
         vm.createSelectFork(env.RPC_URL);
         initializeFromDeployment(env.DEPLOY_CONFIG);
         deployParams = parseDeployParams(env.DEPLOY_CONFIG);
+        adminsCount = block.chainid == 1 ? 1 : 2;
```
Comment (skhomuti) [inline test/fork/deployment/PostDeployment.t.sol:34]: you can get it by cheching the zero address of `deployParams.secondAdminAddress`

Comment (skhomuti) [reply test/fork/deployment/PostDeployment.t.sol:34]: but chainId is also ok

File: test/fork/invariant/Invariants.t.sol [hunk 2]
```diff
@@ -219,3 +221,13 @@ contract CSFeeOracleInvariant is InvariantsBase {
         );
     }
 }
+
+contract HasConsensusInvariant is InvariantsBase {
```
Comment (skhomuti) [inline test/fork/invariant/Invariants.t.sol:225]:
```suggestion
contract HashConsensusInvariant is InvariantsBase {
```

**Review by skhomuti — CHANGES_REQUESTED** [review]

File: test/fork/deployment/PostDeployment.t.sol (hunk 1 above)
Comment (dgusakov) [reply test/fork/deployment/PostDeployment.t.sol:34]: Let's leave it for now

**Review by skhomuti — APPROVED** [review]

**Review by madlabman — APPROVED** [review]

**Pull request merged**